### ISSUE TITLE
Publish this onto the GetIt Package Manager please 🙏🏻

### ISSUE THREAD

**Issue by shaunroselt** [paolo-rossi/delphi-jose-jwt#64]
https://getitnow.embarcadero.com/

This would be a great addition to GetIt 👌🏻

You can submit it here: https://getitnow.embarcadero.com/submit/

**Comment by fastbike** [comment]
Yes, I'd like to see this more widely available. GetIt looks good.

**Comment by viniciussanchez** [comment]
Try using the Boss. Dependency manager for Delphi

**Comment by shaunroselt** [comment]
> Try using the Boss. Dependency manager for Delphi

I personally have zero interest in using Boss or any third party dependency/package manager when RAD Studio already has a built-in package manager.

I think it would be great if this library can be added to GetIt ❤️

**Comment by paolo-rossi** [comment]
Hello Shaun, David... 

thank you for your suggestion, I will take the time to submit JWT (and Neon) to GetIt seeing that is the default "package manager" shipped with Delphi.

I must say that I consider Boss (thanks Vinicius) far superior compared to GetIt but... we can have both, right? :-)

Ciao,
Paolo.

**Comment by sbeu** [comment]
I looked into the Getit package manager and I did not see any way to handle the versions of the packages. 
Is GetIt as flexible as boss, regarding the packages versions?
